web/src/components/filings-calendar/tax-access/TaxAccess.test.tsx

```diff
@@ -16,19 +17,41 @@ import {
   generateUserDataForBusiness,
 } from "@businessnjgovnavigator/shared/test";
 import { Business } from "@businessnjgovnavigator/shared/userData";
-import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { fireEvent, render, RenderResult, screen, waitFor } from "@testing-library/react";
+
+function setupMockAnalytics(): typeof analytics {
+  return {
+    ...jest.requireActual("@/lib/utils/analytics").default,
+    event: {
+      ...jest.requireActual("@/lib/utils/analytics").default.event,
+      tax_calendar: {
+        arrive: {
+          arrive_calendar_access_v2: jest.fn(),
+        },
+        click: {
+          click_calendar_access_v2: jest.fn(),
+        },
+        submit: {
+          tax_calendar_validation_error: jest.fn(),
+        },
+      },
+    },
+  };
+}
 
 const Config = getMergedConfig();
 jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
 jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+jest.mock("@/lib/utils/analytics", () => setupMockAnalytics());
+const mockAnalytics = analytics as jest.Mocked<typeof analytics>;
 
-describe("<TaxAccessModal />", () => {
-  const renderModal = (business?: Business): void => {
-    render(
+describe("<TaxAccess />", () => {
+  const renderComponent = (business?: Business): RenderResult => {
+    return render(
```

Comment:
pebble - we don't need to return the results of this function. We are not storing the return value from this function. Methods like `screen` can still access the rendered DOM